device/common/include/traccc/finding/device/find_tracks.hpp

```diff
@@ -43,22 +45,24 @@ namespace traccc::device {
 /// @param[out] n_total_candidates  The number of total candidates for the
```

Comment:
Please update the description of the parameters.

Reply:
Done.